src/wms/src/main/java/org/geoserver/wms/capabilities/Capabilities_1_3_0_Transformer.java

```diff
@@ -682,10 +682,14 @@ private void handleLayers() {
             }
             handleRootCrsList(srs);
 
-            CloseableIterator<LayerInfo> layers;
-            layers = catalog.list(LayerInfo.class, filter);
+            // create layer groups filter
+            final Filter lgFilter = Predicates.acceptAll();
```

Comment:
I believe so, yes, the filtering should be uniform. When originally implemented workspace specific groups did not exist, and when they were added, this bit was forgotten about.